tests/phpunit/elementor/core/common/modules/connect/apps/test-common-app.php

```diff
@@ -529,4 +529,39 @@ private function get_connected_user() {
 
 		return $user;
 	}
+
+	public function test_ge111t_remote_authorize_url() {
```

Comment:
Is there not already a test for this?

https://github.com/elementor/elementor/blob/2c4254606e62d9ec6ffa644f947d4c0a49ffe4b0/tests/phpunit/elementor/core/common/modules/connect/apps/test-common-app.php#L479C18-L503

Reply:
@KingYes